lang/fr/strings.po

```diff
@@ -92,23 +92,23 @@ msgstr "boxeur"
 
 #: ../../script/engine.js:16
 msgid "punches do more damage"
-msgstr "les points font plus de dégats"
+msgstr "les coups points font plus de dégats"
 
 #: ../../script/engine.js:18
 msgid "learned to throw punches with purpose"
-msgstr "appris à donner des coups de poings avec un objectif"
+msgstr "vous avez appris à donner des coups de poings avec un objectif"
 
 #: ../../script/engine.js:21
 msgid "martial artist"
 msgstr "maître d'un art martial"
 
 #: ../../script/engine.js:22
 msgid "punches do even more damage."
-msgstr "les coups de poings font plus de dégats"
+msgstr "les coups de poings font encore plus de dégats"
 
 #: ../../script/engine.js:23
 msgid "learned to fight quite effectively without weapons"
-msgstr "apprendre à se battre de façon efficace sans arme"
```

Comment:
In the English text, we drop personal pronouns for a more terse style. Definitely shouldn't be using "vous" in French.

Reply:
Make sense, I didn't knew when I did my translations that's why I used 'vous' in this sentence because it sounded better to me.